src/NServiceBus.Transport.Msmq.AcceptanceTests/When_enabling_journaling.cs

```diff
@@ -22,10 +22,10 @@ public async Task Verify_journaling_flag(bool global)
                 await Scenario.Define<Context>()
                 .WithEndpoint<Endpoint>(b => b.CustomConfig(c =>
                 {
-                    var t = c.UseTransport<MsmqTransport>();
+                    var t = (MsmqTransport)c.ConfigureTransport();
```

Comment:
Would a `c.ConfigureTransport<MsmqTransport>()` be an option to avoid the ugly cast?

Reply:
yes, I'm planning to add that to the testing framework, some other downstreams have used their own extension method to do that.

Reply:
here you go: https://github.com/Particular/NServiceBus/pull/5910 😁